central/clusterinit/service/service_impl.go

```diff
@@ -9,16 +9,46 @@ import (
 	"github.com/stackrox/rox/central/clusterinit/backend"
 	"github.com/stackrox/rox/central/clusterinit/store"
 	v1 "github.com/stackrox/rox/generated/api/v1"
-	"github.com/stackrox/rox/pkg/defaults/accesscontrol"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/grpc/authz"
+	"github.com/stackrox/rox/pkg/grpc/authz/perrpc"
 	"github.com/stackrox/rox/pkg/grpc/authz/user"
+	"github.com/stackrox/rox/pkg/sac/resources"
 	"github.com/stackrox/rox/pkg/set"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 var (
-	authorizer = user.WithRole(accesscontrol.Admin)
+	authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
+		user.With(
+			permissions.View(resources.Administration),
+			permissions.View(resources.Integration),
```

Comment:
This is more of a open-ended question and not necessarily something we may fix in this PR: why do we require `Integration` for _reading_ the CA that's used to sign the init bundles? It's not necessarily tied to a specific init bundle. Theoretically, we could get away with only `Administration` here, but it's nothing we need to change for the time being.